package/yast2-firstboot.spec

```diff
@@ -15,14 +15,8 @@
 # Please submit bugfixes or comments via http://bugs.opensuse.org/
 #
 
-
-#Compat macro for new _fillupdir macro introduced in Nov 2017
-%if ! %{defined _fillupdir}
-  %define _fillupdir /var/adm/fillup-templates
```

Comment:
This was added in previous commit just to be conservative, isn't it?

Reply:
Yes, but it is not needed as `master` won't build in older openSUSE/SUSE versions (we have `SLE-12-SP3`, `SLE-12-SP2`,... Moreover, it points to the old (and wrong) path.